src/Xamarin.Android.Build.Tasks/Utilities/LlvmIrGenerator/LlvmIrStringEncoding.cs

```diff
@@ -0,0 +1,7 @@
+namespace Xamarin.Android.Tasks.LLVMIR;
+
+enum LlvmIrStringEncoding
+{
+	UTF8,
+	Unicode,
```

Comment:
Silly Q, but should we use `UTF16LE` instead of `Unicode`, seeing as "Unicode" doesn't necessarily mean anything…

Reply:
That's what the managed land calls the encoding, so I followed suit.